code/style.css

```diff
@@ -1,4 +1,157 @@
+body{
+  margin:0;
+}
 html {
-  background: #0025ff;
-  color: #fff;
+  color: black;
+  font-family: Helvetica, sans-serif, Arial,;
+  font-size: 12px;
+  background-color: white;
+  
+}
+
+.logo {
```

Comment:
Just a little tip to make the css a bit more clear to read is to be consistent in the spacing . An example:

header {
xxx
}


or

header{
xxx

}